compiler/ballerina-backend-jvm/src/main/ballerina/src/compiler_backend_jvm/interop/external_method_gen.bal

```diff
@@ -192,6 +192,12 @@ function createOldStyleExternalFunctionWrapper(bir:Function birFunc, string orgN
     addDefaultableBooleanVarsToSignature(birFunc);
     bir:BInvokableType functionTypeDesc = birFunc.typeValue;
 
+    bir:BType? restType = functionTypeDesc?.restType;
+
+    if (!(restType is ())) {
```

Comment:
I feel it would increase readability if you use restType is bir:BType instead of the negated statement? WDYT
```suggestion
    if ( restType is bir:BType ) {
```

Reply:
Please see above comment.